recipe/pixi.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "repodata-patches"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
+channels = ["conda-forge", "nodefaults"]
+platforms = ["osx-arm64", "linux-64", "win-64"]
+
+[tasks]
+show-diff = "python ./show_diff.py"
```

Comment:
as this is:
- _very_ expensive, I recommend adding a `description` noting that
- _very_verbose, probably also a task which writes it out to a `> {git branch}.txt`

Reply:
I added the descriptor. I didn't commit the branch name addition as it would complicate the readme and .gitignore.